src/utils.js

```diff
@@ -5,7 +5,44 @@ import getLogger from 'loglevel-colored-level-prefix'
 const logger = getLogger({prefix: 'prettier-eslint'})
 
 /* eslint import/prefer-default-export:0 */
-export {getOptionsForFormatting}
+export {getOptionsForFormatting, defaultEslintConfig}
+
+function defaultEslintConfig(eslintConfig = {}, defaultConfig = {}) {
```

Comment:
Why don't we simply this considerably by using [`lodash.merge`](https://www.npmjs.com/package/lodash.merge)? Could probably reduce the amount of tests required by a lot :)